l1-contracts/contracts/dev-contracts/test/ExecutorTest.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+
+import {ExecutorFacet} from "../../state-transition/chain-deps/facets/Executor.sol";
+
+pragma solidity 0.8.24;
+
+contract TestExecutor is ExecutorFacet {
+    /// @dev Since we don't have access to the new BLOBHASH opecode we need to leverage a static call to a yul contract
```

Comment:
please mention that it is only the case in the testing environment

Reply:
And I think the only reason for this is to have correct test coverage. Is this correct?

Reply:
yup